CheckYourEligibility-Admin/Views/Shared/_Layout.cshtml

```diff
@@ -27,6 +26,27 @@
 </head>
 
 <body class="govuk-template__body  js-enabled" data-clarity="@clarityId">
+    <div id="cookie-banner" class="govuk-cookie-banner nojshide" role="region" aria-label="Cookies on Check a family's eligibility" data-nosnippet>
```

Comment:
With this implementation, is there always a cookie flicker whenever you've agreed/rejected cookies and move to different pages?